### ISSUE TITLE
Library doesn't have some Away3d classes

### ISSUE THREAD

**Issue by nikitablack** [mziwisky/bullet-ane#2]
Hello. Your compiled library doesn't have some classes. For example, away3d.materials.lightpickers.StaticLightPicker is missing. And many others. I tried to build aneBulletLib by myself too and got the same results.
Next, aneBulletDefault is not compiled at all. It fails on ane.bulletphysics.collision.dispatch.CollisionWorld in contactTest() method. That's because awayphysics.collision.dispatch.AWPCollisionWorld doesn't have contactTest() method. And there're tons of other errors.

**Comment by mziwisky** [comment]
Ideally, the ANE wouldn't contain _any_ away3d classes, rather it would depend on away3d being simultaneously linked into the app.  However, I haven't figured out a configuration to allow for this.  aneBulletDefault depends on away3d and awayphysics; if I set those as externally-linked dependencies, then I get VerifyErrors in the AIR simulator.  I can't tolerate this, as AIR sim is the fastest way to debug my apps.  Note that iOS sim seems to work with this config, and I can't speak about actual hardware because I don't have any to test on.  Anyway, to get AIR sim working, I link said libraries as merged-into-code.  I do the same for aneBulletLib's dependency on away3d.  Unfortunately, this means that the away3d classes that these libs use (and _only_ those classes) will become linked into the ANE.  If you also link away3d into your app, it seems to work fine, but only if it's the same away3d codebase that got linked into the ane libs.  If they differ, you might get VerifyErrors.

As for the missing contactTest() method, you'll have to get that from **the dev branch** of my fork of awayphysics:  https://github.com/mziwisky/awayphysics-core-fp11

I probably should have put that tip in the README.

Ultimately, it would be nice to strip bullet-ane of its dependency on away3d and awayphysics, and allow the user to plug in their 3d framework of choice.  I'd like to work on this, but this entire project is a very low priority in my life right now.  I'd love to have some contributors work towards this end.

In any event, drop a line here to let me know if you have success building on your machine.  If so, I'll close this issue; if not, I'll try to help you out further.

**Comment by nikitablack** [comment]
Hi. Thank you. It was my mistake - looking into library.swf of ane I thought that it contain away3d. I just add the whole source code for away3d and can run your .ane on my Windows and on android device. Unfortunately all this stuff is unusable (. Although as3 part runs much faster with ane it suffers from huge memory leaks. And the problem that it's impossible to debug that easy - scout profiler even not show where this memory leaks occurs.

**Comment by mkfui** [comment]
sir, error message:
TypeError: Error #1009: Cannot access a property or method of a null object reference.
	at ane.bulletphysics.dynamics::DiscreteDynamicsWorld()
	at AneGravityTest/init()
	at flash.display::DisplayObjectContainer/addChild()
	at aneBulletTester()
i have add to library awayphysic but still got error, if only run awayphysic example, ok but bullet example error occur

**Comment by mziwisky** [comment]
Sorry, I can no longer provide maintenance on this library.
